ftxwebsocket/client.py

```diff
@@ -7,20 +7,21 @@
 from typing import DefaultDict, Deque, List, Dict, Tuple, Optional
 from gevent.event import Event
 
-from websocket.websocket_manager import WebsocketManager
+from ftxwebsocket.websocket_manager import WebsocketManager
 
 
 class FtxWebsocketClient(WebsocketManager):
     _ENDPOINT = 'wss://ftx.com/ws/'
 
-    def __init__(self) -> None:
+    def __init__(self, on_disconnect: callable = None) -> None:
```

Comment:
Introduce a disconnection callback that fires when a websocket connection is lost (e.g. on error / closure). This is used to trigger a task that fetches any records missed between websocket connection loss and restoration via REST API requests.